### ISSUE TITLE
Popular Beatmaps should forcefully include non osu! gamemodes

### ISSUE THREAD

**Issue by nl-tatatat** [ppy/osu-web#5514]
For as long as I've been playing, I've never seen a non-osu! beatmap in the most played/popular beatmap section. I don't think that's representative of what the whole community considers popular. Many times non-osu! beatmaps will be very popular in their respesctive, but will never reach the popular beatmaps section because the other gamemodes simply don't have as big of a player-base as osu!standard I think including other gamemodes in the Popular Beatmaps section would help promote these other gamemodes.

I have a couple solutions for this.

 1. Force at least one of the Popular Beatmaps slots to always be a non-osu! gamemode.

    This is in my opinion the best option. It ensures that a non-osu! beatmap will always be in the popular beatmaps, no matter what. The problems I see with it though is that its an arbitrary amount of slots, and it might run into the same issue its trying to fix: one of the non-osu! gamemodes might be popular enough to where the other two gamemodes rarely if ever reach the Popular Beatmaps section.

 2. Weight non-osu! gamemodes more favorably in the Popular Beatmaps formula.

    This solves the issue of one of the gamemodes being more popular than the others and never allowing them to enter the popular beatmaps section, but it seems like a very finicky solution. What is the right ratios and formulas for weighting? I sure don't know.

3. Add separate tabs for non-osu! gamemodes

    This could be integrated nicely with player's default gamemode on their userpage, always showing the gamemode they are interested in. A lot of non-osu! players aren't invested are interested in osu! beatmaps. Only their own gamemode. The problem I see with this is that hybrid beatmaps with osu! diffs are bound to get tons more plays.

**Comment by venix12** [comment]
third solution sounds the best i think

**Comment by nl-tatatat** [comment]
Is candidate Issues being deprecated? 
I don’t understand.

**Comment by peppy** [comment]
Yes, we changed the way re project management. Some details here: https://github.com/ppy/osu/wiki/Project-management
